core/config/fungibleConfig.js

```diff
@@ -0,0 +1,23 @@
+import harden from '@agoric/harden';
+
+import { noCustomization } from './noCustomization';
+import { makeCoreMintKeeper } from './coreMintKeeper';
+import { makeNatAssay } from './assays';
+
+// Fungible tokens (our default for mints) do not customize
```

Comment:
This text implies that we will never want to create more elaborate fungible tokens, i.e., that this will never be the base for further customizations. From the meter and fractal reserve bank, we already know that we will.

Rather, this should be described as basic fungible or core fungible or something, to fit with your overall terminology conventions for these inheritance relationships.